src/desktop/apps/auction/components/layout/auction_info/Registration.js

```diff
@@ -5,6 +5,33 @@ import { get } from "lodash"
 import { connect } from "react-redux"
 import { Button, Sans } from "@artsy/palette"
 
+function RegistrationText(props) {
+  const {
+    userNeedsIdentityVerification,
+    defaultText = "Registration required to bid",
+    defaultColor = "black60",
+  } = props
+  if (userNeedsIdentityVerification) {
+    return (
+      <Sans mt="1" size="3" color="black60" textAlign="center">
+        Identity verification required to bid.{" "}
+        <a
+          target="_blank"
+          href="https://www.artsy.net/identity-verification-faq"
```

Comment:
Minor: Can scope this to the force environment that the user is in? 

```suggestion
          href="/identity-verification-faq"
```

We do something similar below:

https://github.com/artsy/force/blob/06a319844b18be07492b5494c4581594d9ca8fc9/src/desktop/apps/auction/components/layout/auction_info/Registration.js#L111